10up/cypress-wp-utils
# Command/40 Added `checkBlockPatternExists` command.

### Description of the Change

Added a new command `checkBlockPatternExists` which checks whether a given pattern exists on the site or not.

It accepts the following parameters.

* `title`: Pattern name/title.
* `categoryValue`: Value of the pattern category. for example "twentytwentyone" for the Twenty Twenty-One category.


Closes #40

### Verification Process

1. Build the project using `npm run build`.
2. Make sure the docker is running.
3. Run `npm run env:start`.
4. Run `npm run cypress:open`.
5. Run `check-block-pattern-exists.test.js`.
6. Try to change the pattern name and its category in the `/tests/cypress/integration/check-block-pattern-exists.test.js` file and see the test results.

### Checklist:






### Changelog Entry

> Added - New Command `checkBlockPatternExists` to check if block pattern exists.

### Credits


Props @faisal-alvi
